Doc/library/threading.rst

```diff
@@ -285,7 +285,7 @@ thread of control.  There are two ways to specify the activity: by passing a
 callable object to the constructor, or by overriding the :meth:`~Thread.run`
 method in a subclass.  No other methods (except for the constructor) should be
 overridden in a subclass.  In other words, *only*  override the
-:meth:`~Thread.__init__` and :meth:`~Thread.run` methods of this class.
+``Thread.__init__()`` and :meth:`~Thread.run` methods of this class.
```

Comment:
In the original, both have `~Thread.`, and the tilde means hide the `Thread.`:

![image](https://github.com/python/cpython/assets/1324225/4d6b211d-0a1e-44d4-8e27-1eb084ba96ab)

However, this now shows the first one:

![image](https://github.com/python/cpython/assets/1324225/7f3236b1-db75-48ee-afd0-be1be069ddea)

Shall we keep it consistent?

```suggestion
``__init__()`` and :meth:`~Thread.run` methods of this class.
```